data-extraction/pyDataExtraction/commonTypes/Text.py

```diff
@@ -0,0 +1,64 @@
+from typing import Optional
+from pyDataExtraction.commonTypes.base import DataType
+
+"""
+interface Text {
+	kind: { text: true };
+	text: string;
+	mimeType?: string;
+	fileName?: string;
+}
+"""
+
+
+class Text(DataType):
+    def __init__(
+        self,
+        text_data: str,
+        mimeType: Optional[str] = None,
```

Comment:
let's keep it to the pep-8 python style guide:
```
mime_type,
file_name
```

Reply:
the variable names are mainly because of the use case and the method currently for doing so. 

right now I'm directly converting the respective dataTypes to `json` objects when they are printed or called as a string, and variables are the displayed variables inside the `json` string, so I've been matching the case of the specification listed on the Readme.

Though since we already need to do some manipulations on the json representation before returning in the case of the `Edge` class, doing so here shouldn't be a major issue.

btw if we got rid of the reliance on `__dict__`, we could use `__slots__` to slightly improve performance and reduce memory usage.